test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -633,6 +633,10 @@ def test_pip_check(image):
 
     :param image: ECR image URI
     """
+
+    if "pytorch" in image and "trcomp" in image:
```

Comment:
why is this skipped? Do you mean the pip check will always be broken in the image?

Reply:
yes. pytorch xla lists tpu-client as a runtime dependency but it's not installed.